js/tests/index.html

```diff
@@ -7,7 +7,7 @@
 
     <!-- jQuery -->
     <script src="../../docs/assets/js/vendor/jquery-slim.min.js"></script>
-    <script src="../../docs/assets/js/vendor/tether.min.js"></script>
+    <script src="../../docs/assets/js/vendor/popper.min.js"></script>
```

Comment:
Hi @FezVrasta,

For an unknown reason my unit tests failed because it seems when I run ``grunt qunit`` on my bash Popper is undefined but as you can see I added Popper. 

Do you have any idea ? 

In `tooltip.js` we have this check : 
```javascript
  /**
   * Check for Popper dependency
   * Popper - https://popper.js.org
   */
  if (typeof Popper === 'undefined') {
    throw new Error('Bootstrap tooltips require Popper (https://popper.js.org)')
  }
```
And it's this exception which failed tests